framework/core/js/src/forum/components/UserSecurityPage.tsx

```diff
@@ -51,17 +51,31 @@ export default class UserSecurityPage<CustomAttrs extends IUserPageAttrs = IUser
   settingsItems() {
     const items = new ItemList<Mithril.Children>();
 
-    ['developerTokens', 'sessions'].forEach((section) => {
-      const sectionName = `${section}Items` as 'developerTokensItems' | 'sessionsItems';
-      const sectionLocale = camelCaseToSnakeCase(section);
-
+    if (
+      app.forum.attribute('canCreateAccessToken') ||
+      app.forum.attribute('canModerateAccessTokens') ||
+      (this.state.hasLoadedTokens() && this.state.getDeveloperTokens()?.length)
+    ) {
       items.add(
-        section,
-        <FieldSet className={`UserSecurityPage-${section}`} label={app.translator.trans(`core.forum.security.${sectionLocale}_heading`)}>
-          {this[sectionName]().toArray()}
+        'developerTokensItems',
+        <FieldSet
+          className="UserSecurityPage-developerTokensItems"
```

Comment:
These classes shouldn't contain "Items"

```suggestion
          className="UserSecurityPage-developerTokens"
```